src/libraries/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -528,6 +534,7 @@ public abstract partial class MD5 : System.Security.Cryptography.HashAlgorithm
     [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
     public partial class PKCS1MaskGenerationMethod : System.Security.Cryptography.MaskGenerationMethod
     {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("PKCS1MaskGenerationMethod it not trim compatible because the algorithm implementation referenced by HashName might be removed.")]
```

Comment:
```suggestion
        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("PKCS1MaskGenerationMethod is not trim compatible because the algorithm implementation referenced by HashName might be removed.")]
```

Reply:
This mistake looks repeated. I'll fix them all. Nice catch.